EMBEDS_README.md

```diff
@@ -0,0 +1,21 @@
+# Let's get to know each other a little better
+Welcome (temporarily) to the team!  Although this engagement isn't permanent, let's take this opportunity to set a few expectations.  Many of the tenants from my [Manager README for full-time direct reports](https://github.com/esquireofoz/Manager-README/blob/master/README.md) apply, but the nature of an embedding arrangement have some additional nuances we'll cover here.
```

Comment:
Ahh is it `tenants` or `tenets`? 🤔 :thinky_right:

Reply:
`have some` -> `has some` I _think_ because "nature" is singular

Reply:
I think [tenants](https://en.wiktionary.org/wiki/tenant#English) is correct.  Good catch on grammar for `have` vs. `has`.